build.assets/Dockerfile

```diff
@@ -177,7 +177,7 @@ RUN go install github.com/daixiang0/gci@v0.8.2
 RUN (mkdir -p helm-tarball && curl -L https://get.helm.sh/helm-v3.5.2-$(go env GOOS)-$(go env GOARCH).tar.gz | tar -C helm-tarball -xz && \
      cp helm-tarball/$(go env GOOS)-$(go env GOARCH)/helm /bin/ && \
      rm -r helm-tarball*)
-RUN helm plugin install https://github.com/vbehar/helm3-unittest && \
+RUN helm plugin install https://github.com/quintush/helm-unittest && \
```

Comment:
Can we introduce a Makefile target that installs this Helm plugin as a dependency to all targets that require it? This way folks running locally will get it installed automatically.

Also, can you add a comment here explaining why we're using this particular fork of the helm-unittest and not some other.

Reply:
I'll add the fork explanation in the chart README, we have a section dedicated to tests.

I'll also add the auto-install target in the PR chaging the plugin we use. Auto-installing in this PR will cause the wrong plugin version to be installed.

Reply:
I addressed your comments here: https://github.com/gravitational/teleport/pull/18964